src/components/App.tsx

```diff
@@ -918,6 +918,19 @@ class App extends React.Component<AppProps, AppState> {
   }
 
   private onResize = withBatchedUpdates(() => {
+    if (!this.device.isMobile) {
+      const scrollBarWidth = 10;
+      const widthRatio =
+        (window.outerWidth - scrollBarWidth) / window.innerWidth;
+      const isBrowserZoomed =
+        // Device pixel ratio is 2 when zoom is 100%
+        window.devicePixelRatio !== 2 || widthRatio < 0.9 || widthRatio > 1;
```

Comment:
`window.devicePixelRatio !== 2` will trigger on any resize on screens that don't have dPR === 2.